Tone/signal/Pow.ts

```diff
@@ -0,0 +1,85 @@
+import { Signal, SignalOptions } from "./Signal";
+import { WaveShaper, WaveShaperMappingFn } from "./WaveShaper";
+import { optionsFromArguments } from "Tone";
+
+/**
+ *  @class Pow applies an exponent to the incoming signal. The incoming signal
+ *         must be AudioRange.
+ *
+ *  @constructor
+ *  @param exp The exponent to apply to the incoming signal, must be at least 2. 
+ *  @example
+ * var pow = new Pow(2);
+ * var sig = new Signal(0.5).connect(pow);
+ * //output of pow is 0.25. 
+ */
+export class Pow extends Signal<number> {
+	
+	readonly name: string = "Pow";
+
+	/**
+	 * Indicates if the value should be overridden on connection
+	 */
+	readonly override = false;
```

Comment:
This probably isn't necessary. Override is a flag which indicates if a value should be overwritten when it is connected to, but since it's not a schedulable value like Signal or Multiply, there is nothing to override. the signal is just processed as it passes through the unit.

Reply:
Thanks for the explanation, I did not realize that at first but it does make sense looking at it again